core/modules/wmgr/python/dbMgr.py

```diff
@@ -517,17 +588,42 @@ def createChunk(dbName, tblName):
     _log.debug('will create tables: %s', tables)
     for tblType, chunkTable in tables.items():
 
-        # make table using DDL from non-chunked one
-        query = "CREATE TABLE {2}.{0} LIKE {2}.{1}".format(chunkTable, tblName, dbName)
-        _log.debug('make chunk table: %s', chunkTable)
-        try:
+        # check if this table is a ctually a view
+        if dbConn.isView(tblName, dbName):
+            
+            # view needs more complicated algorithm to copy its definition, first copy
+            # its current definition, then rename existing view and then re-create it again
```

Comment:
OK, however it seems the view still point on the (empty) XXX table whereas it should point on the XXX_chunkId table, isn't it? Here we have to guess XXX and rename it XXX_chunkId, isn't it?